### PR TITLE
RSDK-2003 Move AttrConf from builtin.go to slam

### PR DESCRIPTION
**Pull request by zaporter-work**
Ticket: https://viam.atlassian.net/browse/RSDK-2003

This should only be done after https://github.com/viamrobotics/slam/pull/159

### CODE REVIEW

**Review by kkufieta — APPROVED** [review]
LGTM mod nit

File: services/slam/builtin/builtin.go [hunk 1]
```diff
@@ -53,8 +52,8 @@ var (
 )
 
 const (
-	defaultDataRateMs           = 200
-	minDataRateMs               = 200
+	defaultDataRateMsec           = 200
+	minDataRateMsec               = 200
```
[the remark this answers is not in the document]
Comment (nicksanford) [reply services/slam/builtin/builtin.go:56]: Also, should `minDataRateMsec` still be in builtin.go?

Your PR to move code into the slam repo appears to not use it: https://github.com/viamrobotics/slam/pull/159#pullrequestreview-1327281919

**Review by nicksanford — CHANGES_REQUESTED** [review]

**Review by nicksanford — APPROVED** [review]
LGTM as long as we know whether or not we should remove `minDataRateMs` or not.

**Review by tessavitabile — APPROVED** [review]
Yes, in https://github.com/viamrobotics/slam/pull/159, I said it's fine to remove the cap on data rate, since it's doing more harm than good. Zack resolved the ticket here: https://viam.atlassian.net/browse/RSDK-1922

**Pull request merged**